### PR TITLE
specify what osm-coordinate means

### PR DESCRIPTION
**Pull request by SlowMo24**
### Description
Specifies what osm-coordinates are.

### Checklist
- [x] My code follows the [code-style](https://github.com/GIScience/oshdb/blob/master/CONTRIBUTING.md) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/oshdb/view/change-requests/) and [benchmark](https://reports.ohsome.org/oshdb-benchmarks/) (if applicable) results
 - ~I have commented my code~
- [X] I have written javadoc (required for public classes and methods)
- ~I have added sufficient unit tests~
- ~I have made corresponding changes to the [documentation](https://github.com/GIScience/oshdb/tree/master/documentation)~
- ~I have updated the [CHANGELOG.md](https://github.com/GIScience/oshdb/blob/master/CHANGELOG.md)~
- ~I have adjusted the [examples](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-examples) or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-examples/-/issues/new) in the corresponding repository~
- ~I have adjusted the [benchmark](https://reports.ohsome.org/oshdb-benchmarks/) or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-benchmarks/-/issues/new) in the corresponding repository~

### CODE REVIEW

File: oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java [hunk 1]
```diff
@@ -24,7 +24,11 @@
   private final int maxLat;
 
   /**
-   * Creates an {@code OSHDBBoundingBox} instance from osm int coordinates.
+   * Creates an {@code OSHDBBoundingBox} instance from transposed coordinates.
+   * 
+   * This method is mainly for internal usage. Coordinates are stored as integer
```
Comment (tyrasd) [inline oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java:29]: a `<p>` is needed here for correct javadoc formatting:

```suggestion
   * <p>This method is mainly for internal usage. Coordinates are stored as integers
```

File: oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java [hunk 2]
```diff
@@ -24,7 +24,11 @@
...
+   * internally by multiplying and rounding them (see 
+   * {@code bboxWgs84Coordinates}) named osm-coordinates.
```
Comment (tyrasd) [inline oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java:31]:
```suggestion
   * Creates an {@code OSHDBBoundingBox} instance from scaled OSM coordinates.
   * 
   * <p>This method is mainly for internal usage. OSM stores coordinates with a fixed precision
   * of <a href="https://wiki.openstreetmap.org/wiki/Node#Structure">7 decimal digits</a> and
   * stores them internally as integers. You can use this method to create a bounding box from
   * such (scaled and rounded) coordinates.</p>
```

**Review by tyrasd — COMMENTED** [review]
maybe use the term _scaled_ instead of _transposed_?

File: oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java (hunk 2 above)
Comment (SlowMo24) [reply oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java]: thanks, this is much better, I adjusted it some more

**Comment by SlowMo24** [comment]
@tyrasd feel free to adapt further, for me this is ready.

**Review by tyrasd — DISMISSED** [review]
:+1:

**Review by tyrasd — APPROVED** [review]

**Pull request merged**